platform/event/core_event_data.proto

```diff
@@ -0,0 +1,74 @@
+/**
+ * # Core Event Data
+ * Contains information about an event and its parents.
```

Comment:
```suggestion
 * A message that describes the metadata for an event.
 *
 * The `CoreEventData` links an event and it's list of parent events, and also includes
 * the software version and identifier for the node that created this event as well as the
 * birth round and creation timestamp for the event.
```

Reply:
Added this extra description, with some modifications (in particular, I removed the phrase "links an event", since this expression collides with the concept of event linking that is internal to the platform)